lib/opensusebasetest.pm

```diff
@@ -499,6 +499,9 @@ sub wait_boot {
             #assert_screen "dm-password-input", 10;
             elsif (check_var('DESKTOP', 'gnome')) {
                 # In GNOME/gdm, we do not have to enter a username, but we have to select it
+                if (is_tumbleweed) {
```

Comment:
maybe better just :
`my $key = is_tumbleweed() ? 'tab' : 'ret';`
and `send_key $key;`

Reply:
Thank you but actually tumbleweed needs `tab` then `ret` while others don't need `tab`

Reply:
Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check.

Reply:
Thank you for the comment!
What it occurs to me is that toggling the logic check will add some redundancy to this section, since we might have to write `send_key \'ret\'` two times. Do you have a suggestion to toggle the logic in a comparably beautiful way?

Additionally, what's on my mind is that these test scripts are based on SLE, which is more stable with lower frequency in changing features.
As it sounds to me, Tumbleweed might require more changes over time, are we migrating all test logic in all scripts to make Tumbleweed as our default product now?